tensorflow/compiler/mlir/tools/kernel_gen/transforms/tf_framework_legalize_to_llvm.cc

```diff
@@ -319,7 +321,7 @@ class JITCompileFromStrOpConverter
         llvm::makeArrayRef({adaptor.ctx(), jit_module_code, tile_sizes.first,
                             tile_sizes.second, unroll_factors.first,
                             unroll_factors.second, max_supported_rank,
-                            enable_ftz, cpu_codegen}));
+                            enable_ftz, index_64bit, cpu_codegen}));
```

Comment:
Can we have a test for this in `tensorflow/compiler/mlir/tools/kernel_gen/tests/tf_framework_legalize_to_llvm.mlir`?

Reply:
test modified to include the parameter.